### PR TITLE
Updated `expand` to show full body when clicking "Show More"

### PR DESCRIPTION
**Pull request by jfeigel**

### CODE REVIEW

File: app/directives/expand/expand.directive.js [hunk 1]
```diff
@@ -27,9 +29,20 @@ var Expand = function($compile, $timeout) {
             scope.showExpandLink = newVal && scope.needExpandLink;
           });
 
-          scope.onExpandClick = function() {
+          scope.onExpandClick = function(e) {
```
Comment (albi34) [inline app/directives/expand/expand.directive.js]: Do we need the $event parameter here ? I'm not seeing it being used in the function ?

File: app/directives/expand/expand-mouseenter.template.pug [hunk 2]
```diff
@@ -1,6 +1,6 @@
 div(ng-mouseenter="hover = true" ng-mouseleave="hover = false")
-  div.expandNoHover(ng-bind-html="item" ng-attr-style='-webkit-line-clamp: {{expanded ? "initial" : lines || 2}}' ng-click="click()")
-  a.expandLink(href="" ng-click="onExpandClick()" ng-show="showExpandLink") Show&nbsp;
+  div.expandNoHover(ng-bind-html="textShown" ng-attr-style='-webkit-line-clamp: {{expanded ? "initial" : lines || 2}}' ng-class="{'expanded': expanded}" ng-click="click()")
+  a.expandLink(href="" ng-click="onExpandClick($event)" ng-show="showExpandLink") Show&nbsp;
```
Comment (albi34) [inline app/directives/expand/expand-mouseenter.template.pug]: Are we using $event ?

File: app/directives/expand/expand.directive.js [hunk 3]
```diff
@@ -27,9 +29,20 @@ var Expand = function($compile, $timeout) {
...
             scope.showExpandLink = scope.expanded ? false : true;
             scope.expanded = !scope.expanded;
+            if (scope.expanded === true && scope.itemFull !== undefined) {
```
Comment (dominiksteiner) [inline app/directives/expand/expand.directive.js]: can we simply write it as
```
if(scope.expanded && scope.itemFull)
```

File: app/directives/expand/expand.directive.js [hunk 4]
```diff
@@ -27,9 +29,20 @@ var Expand = function($compile, $timeout) {
...
+              scope.textShown = scope.itemFull;
+            } else {
+              scope.textShown = scope.item;
+            }
+
+            if (attrs.hover !== undefined) {
```
Comment (dominiksteiner) [inline app/directives/expand/expand.directive.js]: can we simply say
```
if(attrs.hover)
```

**Comment by jfeigel** [comment]
@albi34 @dominiksteiner I made the changes you both requested.

**Pull request merged**